K2Bridge/KustoDAL/KustoResponseParser.cs

```diff
@@ -180,7 +180,7 @@ private void ReportQueryExecutionMetrics(KustoResponseDataSet kustoResponseDataS
                         createBucketFromDataRow = (DataRow row) => BucketFactory.CreateTermsBucketFromDataRow(row);
                         break;
                     case nameof(Models.Request.Aggregations.DateHistogramAggregation):
-                        createBucketFromDataRow = (DataRow row) => BucketFactory.CreateDateHistogramBucketFromDataRow(row);
+                        createBucketFromDataRow = (DataRow row) => BucketFactory.CreateDateHistogramBucketFromDataRow(row, Logger);
```

Comment:
+1 on having more logs
but it needs to be consistent, in line 180 for example we dont pass this logger. 
and I think in other places we create a new logger and dont pass it around...